### PR TITLE
Refactor Set 2 of React Components

### PR DESCRIPTION
**Pull request by diboy2**

### CODE REVIEW

File: client/src/ZoomToggleButton.jsx [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-    render() {
-        return (
-            <button onClick={this.props.paginator.toggleZoom}>
-                {this.props.paginator.state.zoom ? "Turn Zoom Off" : "Turn Zoom On"}
-            </button>
-        );
-    }
-}
+const ZoomToggleButton = ({ paginator }) => {
+    return <button onClick={paginator.toggleZoom}>{paginator.state.zoom ? "Turn Zoom Off" : "Turn Zoom On"}</button>;
```
Comment (demiankatz) [inline client/src/ZoomToggleButton.jsx]: Extremely trivial point, but we could make the line shorter by factoring "Turn Zoom" out of the two strings and only making Off/On conditional.

**Review by demiankatz — COMMENTED** [review]
Apart from the problem with thumbnail scrolling we're discussing in Teams, the only comment I had was very minor. I haven't thoroughly reviewed the Thumbnail and PaginatorList components yet since I expect they may change while this bug is fixed.

File: client/src/ZoomToggleButton.jsx (hunk 1 above)
Comment (diboy2) [reply client/src/ZoomToggleButton.jsx]: Made change

**Comment by demiankatz** [comment]
Thanks again, @diboy2 -- this all looks good to me (and works as expected). I imagine there is further work that could be done to simplify the relationship between the PaginatorList and Thumbnail components, but this is incremental progress and I assume that will be part of future work on state management.

**Review by crhallberg — APPROVED** [review]
The PaginatorControlGroup refactor in particular is really elegant and instructive for me! Thank you!

**Pull request merged**